src/main/java/com/royalrangers/bean/achievement/UserRewardBean.java

```diff
@@ -0,0 +1,17 @@
+package com.royalrangers.bean.achievement;
+
+import com.royalrangers.model.achievement.Reward;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Date;
+
+@Getter
+@Setter
+public class UserRewardBean {
+    private Long id;
+    private Date createDate;
+    private Date updateDate;
```

Comment:
I have talked with Oleg and he said that on the UI side they don't need this fields. You can leave `createDate` but please remove `updateDate` from this bean.